src/main/java/org/gridsuite/study/server/StudyController.java

```diff
@@ -217,17 +217,18 @@ public ResponseEntity<Void> cutAndPasteNodeSubtree(@PathVariable("studyUuid") UU
         return ResponseEntity.ok().build();
     }
 
-    @PostMapping(value = "/studies/{studyUuid}/tree/subtrees", params = {"subtreeToCopyParentNodeUuid", "referenceNodeUuid"})
+    @PostMapping(value = "/studies/{targetStudyUuid}/tree/subtrees", params = {"subtreeToCopyParentNodeUuid", "referenceNodeUuid"})
     @Operation(summary = "duplicate a subtree")
     @ApiResponses(value = {
         @ApiResponse(responseCode = "200", description = "The subtree was successfully created"),
         @ApiResponse(responseCode = "403", description = "The subtree can't be copied above the root node nor around itself"),
         @ApiResponse(responseCode = "404", description = "The source study or subtree doesn't exist")})
-    public ResponseEntity<Void> duplicateSubtree(@PathVariable("studyUuid") UUID studyUuid,
+    public ResponseEntity<Void> duplicateSubtree(@Parameter(description = "The study where we want to copy the node") @PathVariable("targetStudyUuid") UUID targetStudyUuid,
+                                                 @Parameter(description = "The copied node original study") @RequestParam(value = "sourceStudyUuid", required = false) UUID sourceStudyUuid,
```

Comment:
I think it's better to make both parameter required and call this endpoint with the same value in both parameters when copying subtree in the same study
(`targetStudyUuid` and `sourceStudyUuid`)

Reply:
Done, i thus removed the ternary operation when calling the service method